quarto/03-QC-LBT01.qmd

```diff
@@ -50,55 +54,63 @@ We'll keep the stats for the overall column in a separate data.frame to generali
-                             variables = c(AVAL, CHG), 
-                             statistic = 
-                               list(everything() ~ continuous_summary_fns(c("N", "mean", "sd", "median", "min", "max"))))
+ard_overall <- ard_continuous(adlb,
+  strata = c(PARAMCD, AVISIT), # note arm var is removed
+  variables = c(AVAL, CHG),
+  statistic =
+    list(everything() ~ continuous_summary_fns(c("N", "mean", "sd", "median", "min", "max")))
+) |>
+  apply_fmt_fn() |>
+  unlist_ard_columns()
 ```
 
 ### 4.Statistics comparison
 
 #### A. {rtables} reformat
 Some reformatting is required to ensure the resulting data.frames are compatible. Most of the reformatting is adjusting the column names to match that of the ARD result.
 
-  1.  rename `group3` and `group3_level`, to `group1` and `group1_level` (contains "ARM" values).
-  2.  rename `group1` and `group1_level`, to `group2` and `group2_level` (contains "PARAMCD" values).
-  3.  rename `group2` and `group2_level`, to  `group3` and `group3_level` (contains "AVISIT" values).
-  4.  recode `stat` `n` value to `N` to match ARD
+  1.  Rename `group3` and `group3_level`, to `group1` and `group1_level` (contains "ARM" values).
+  2.  Rename `group1` and `group1_level`, to `group2` and `group2_level` (contains "PARAMCD" values).
+  3.  Rename `group2` and `group2_level`, to  `group3` and `group3_level` (contains "AVISIT" values).
+  4.  Recode `stat` `n` value to `N` to match ARD
   5.  Select columns relevant to comparison
 
 We'll also separate the statistics related to the overall ARM to match the `ard_overall` object.
 
 ```{r}
 #| code-fold: false
-#| code-summary: "Show the code"
 rtables_result <- rtables_result |>
-  dplyr::mutate(stat_name = dplyr::recode(stat_name, "n" = "N"),
-                group4_level = dplyr::recode(group4_level, 
-                                             "Analysis Value" = "AVAL",
-                                             "Absolute Change from Baseline" = "CHG")) |>
-  dplyr::rename(group1 = group3,
-                group1_level = group3_level,
-                group2 = group1,
-                group2_level = group1_level,
-                group3 = group2,
-                group3_level = group2_level,
-                variable = group4_level,
-                # giving a dummy name for this column so it doesn't interfere with the column renaming
-                val = variable) |>
+  dplyr::mutate(
+    stat_name = dplyr::recode(stat_name, "n" = "N"),
+    group4_level = dplyr::recode(group4_level,
+      "Analysis Value" = "AVAL",
+      "Absolute Change from Baseline" = "CHG"
+    )
+  ) |>
+  dplyr::rename(
+    group1 = group3,
+    group1_level = group3_level,
+    group2 = group1,
+    group2_level = group1_level,
+    group3 = group2,
+    group3_level = group2_level,
+    variable = group4_level,
+    # giving a dummy name for this column so it doesn't interfere with the column renaming
+    val = variable
```

Comment:
I am still a bit puzzled by this. Why do you think this differs? We may want to check this at source at a certain point

Reply:
I've changed the ARD creation code to eliminate this renaming.